#!/usr/bin/env python
PACKAGE = "two_d_guidance"
    
from dynamic_reconfigure.parameter_generator_catkin import *
    
gen = ParameterGenerator()
    
gen.add("mask_threshold", int_t,    0, "Mask Threshol", 196,  0, 255)
   
display_mode_enum = gen.enum([ gen.const("BirdEye",  int_t, 0, "Bird Eye"),
                               gen.const("Mask",     int_t, 1, "Mask"),
                               gen.const("Contour",  int_t, 2, "Contour")],
                             "An enum to set display mode")
   
gen.add("display_mode", int_t, 0, "Display mode", 1, 0, 2, edit_method=display_mode_enum)
   
exit(gen.generate(PACKAGE, "two_d_guidance", "fl_lane_detector"))
